core/src/main/scala/org/apache/spark/Accumulator.scala

```diff
@@ -156,5 +165,23 @@ object AccumulatorParam {
     def zero(initialValue: Float): Float = 0f
   }
 
-  // TODO: Add AccumulatorParams for other types, e.g. lists and strings
+  // Note: when merging values, this param just adopts the newer value. This is used only
```

Comment:
This is a little confusing and I maybe wonder whether it'd be worth having a class named something like `LatestValueAccumulableParam[T]` to handle this in a generic way, but for now this is okay.